### ISSUE TITLE
internal server error:  'ascii' codec can't encode character u'\\\\u2018'

### ISSUE THREAD

**Issue by petermolnar** [snarfed/bridgy#692]
trying to send  https://petermolnar.net/falling-by-brad-eide/ to flickr, but brid.gy refuses. I'm using a POST, where the body is `source=https%3A%2F%2Fpetermolnar.net%2Ffalling-by-brad-eide%2F&target=https%3A%2F%2Fbrid.gy%2Fpublish%2Fflickr` to `https://brid.gy/publish/webmention`

~~If I was to do it manually, via the form on brid.gy, it seems to be happy.~~
The online form fails with the same message: "Error: 'ascii' codec can't encode character u'\u2018' in position 226: ordinal not in range(128)".

brid.gy return:
```json
{
"headers": {
	"content-type":"text\/html; charset=utf-8",

	"cache-control":"no-cache",
	"content-encoding":"gzip",
	"x-cloud-trace-context":"91d27abc46a21df88512b49ba4b9c78e",
	"vary":"Accept-Encoding",
	"date":"Tue, 09 Aug 2016 11:54:01 GMT",
	"server":"Google Frontend",
	"content-length":"118",
	"alt-svc":"quic=\":443\"; ma=2592000; v=\"36,35,34,33,32,31,30\""
},
"body":"{
	\n  \"error\": \"Error:  'ascii' codec can't encode character u'\\\\u2018' in position 226: ordinal not in range(128)\"\n }",
"response":{
	"code":500,
	"message":"Internal Server Error"
},
"cookies":[],
"filename":null
}
```

I'm not certain what is going on here.

**Comment by petermolnar** [comment]
The funky part: that page is not supposed to have \u2018 in there.

**Comment by snarfed** [comment]
thanks for the report! looks like it's choking on the smart quote unicode chars in your quoted text. should be an easy fix on my end.

**Comment by snarfed** [comment]
fixed by snarfed/granary@faffe86578ba3b2beae6a6cfebb1c8e9224a6831
